index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Home Page</title>
+		<link rel="stylesheet" href="style.css" />
+	</head>
+	<body>
+		<main>
+			<nav>
+				<a href="index.html">Home</a>
+				<a href="bio.html">Bio</a>
+				<a href="experience.html">Experience</a>
+				<a href="portfolio.html">Portfolio</a>
+			</nav>
+			<article>
+				<div class="text-left">
+					<h1>Welcome to My Portfolio</h1>
+					<h2><span style="color:#281f359e">By:</span> Teanna T</h2>
```

Comment:
nitpick: try to avoid in-line style tags unless absolutely necessary - this styling gets more difficult to reuse and scale with lengthier HTML pages